backend/src/main/java/com/handong/rebon/tag/application/TagService.java

```diff
@@ -0,0 +1,32 @@
+package com.handong.rebon.tag.application;
+
+import com.handong.rebon.exception.tag.TagExistException;
+import com.handong.rebon.tag.domain.Tag;
+import com.handong.rebon.tag.domain.repository.TagRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class TagService {
+
+    private final TagRepository tagRepository;
+
+    @Transactional
+    public Long createTag(String tagName) {
+        validateDuplicateTag(tagName);
+
+        Tag newTag = new Tag(tagName);
+
+        Tag savedTag = tagRepository.save(newTag);
+        return savedTag.getId();
+    }
+
+    private void validateDuplicateTag(String name) {
+        if(tagRepository.existsByName(name)){
+            throw new TagExistException();
+        }
```

Comment:
컨벤션에 맞게 띄어쓰기를 추가하면 좋겠네요!
(맥 기준 단축키 opt + cmd + l)
```suggestion
        if (tagRepository.existsByName(name)) {
            throw new TagExistException();
        }
```

Reply:
앗 네 수정하도록 하겠습니다!!